coala_quickstart/coala_quickstart.py

```diff
@@ -65,7 +63,7 @@ def main():
 
     logging.basicConfig(stream=sys.stdout)
     printer = ConsolePrinter()
-    log_printer = LogPrinter(printer)
+    log_printer = logging.getLogger(__name__)
```

Comment:
This should be just `logging.getLogger(__name__)` , not assinging it into `log_printer`.  Then remove `log_printer` everywhere.

Reply:
hmm... why should we remove `log_printer`? we can just leave it for later.

Reply:
is it difficult to remove?  replace with `None` everywhere?

The objective is to remove the unused concept as much as possible.

If you find this is difficult, I am happy to accept it wont be done in this task. (Satwik might disagree tho)

Reply:
@jayvdb What do you mean
>This should be just `logging.getLogger(__name__)`

I don't understand what you ask to do.
Do you mean I should replace uses of log_printer to `logging.getLogger(__name__)`?
like this? https://github.com/seeeturtle/coala-quickstart/blob/74ebc69068008b95746fd28962535bd945978eef/coala_quickstart/coala_quickstart.py#L82 `log_printer -> logging.getLogger(__name__)

Reply:
Replace

```diff
-log_printer = logging.getLogger(__name__)
+logger = logging.getLogger(__name__)
```

Reply:
I see thanks! :smile:

Reply:
you mean to assign `None` into `log_printer`, right?
or do you mean to remove `log_printer` and put `None` to every uses?